src/Native/pkg/runtime.native.System.Data.SqlClient.sni/win/runtime.native.System.Data.SqlClient.sni.pkgproj

```diff
@@ -1,23 +1,21 @@
 ﻿<?xml version="1.0" encoding="utf-8"?>
 <Project ToolsVersion="12.0" DefaultTargets="Build" xmlns="http://schemas.microsoft.com/developer/msbuild/2003">
   <Import Project="$([MSBuild]::GetDirectoryNameOfFileAbove($(MSBuildThisFileDirectory), dir.props))\dir.props" />
-  
   <PropertyGroup>
-    <Version>4.0.1</Version>
+    <Version>4.0.2</Version>
     <!-- use the same naming convention as a runtime package, but don't treat as a runtime dependency -->
     <IdPrefix>runtime.win7-$(PackagePlatform).</IdPrefix>
     <PackagePlatforms>x64;x86;</PackagePlatforms>
     <SkipValidatePackage>true</SkipValidatePackage>
     <!-- don't package this as runtime specific -->
-    <PackageTargetRuntime></PackageTargetRuntime>
+    <PackageTargetRuntime>
+    </PackageTargetRuntime>
```

Comment:
This was done by MSBuild when it round-tripped the project in my tool.  I can undo it for now.